### PR TITLE
tests: add the basic unit test

### LINKED ISSUE #89: Design and init the tests framework based on pytest

### PR DESCRIPTION
**Pull request by leyao-daily**
This PR is to fix #89 and it will init the basic unit test based on `pytest` framework. We can continue working on finisht the unit tests refer to test_filedb.py.

### CODE REVIEW

File: cnap/requirements-test.txt
Comment (Hulongyin) [inline cnap/requirements-test.txt:1]: Move this requirements.txt to tests directory?

File: tests/core/test_filedb.py
Comment (Hulongyin) [inline tests/core/test_filedb.py:1]: Add tests path to python code scan action?

File: cnap/requirements-test.txt
Comment (leyao-daily) [reply cnap/requirements-test.txt:1]: Keep the requirements in one location is a good practise, we do not need seperate them into differenct location i think.

File: tests/core/test_filedb.py
Comment (leyao-daily) [reply tests/core/test_filedb.py:1]: This can be done after this PR merged to make the action have valid path to scan

**Review by Hulongyin — APPROVED** [review]
LGTM.

**Pull request merged**